### ISSUE TITLE
[Linux] Offline messages of ex-friends bug

### ISSUE THREAD

**Issue by YourLocalCultLeader** [ValveSoftware/steam-for-linux#4112]
If someone sends me an offline message and then deletes me from their contact list, when I login and click on the green icon in Steam's large mode (in order to open the offline messages) their name is displayed as "[unknown]" and the text message is not displayed. Opening their profile via the drop-down menu works fine.

Blocking them also works, but the green icon stayed. Opening the offline messages again (via the green icon) showed a popup, which I accepted (it unblocked them), then I was able to see the name and the text message, and the green icon disappeared.

Not a Steam Beta bug, happened about half a year ago with a non-beta Steam, too.

If more information is needed I could try to reproduce this bug myself, with a second steam account.

```
Processor Information:
    Vendor:  AuthenticAMD
    CPU Family:  0x10
    CPU Model:  0x4
    CPU Stepping:  0x3
    CPU Type:  0x0
    Speed:  3411 Mhz
    4 logical processors
    4 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Unsupported
    SSE4a:  Supported
    SSE41:  Unsupported
    SSE42:  Unsupported

Network Information:
    Network Speed:  

Operating System Version:
    Ubuntu 12.04.5 LTS (64 bit)
    Kernel Name:  Linux
    Kernel Version:  3.2.0-93-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11103000
    X Window Manager:  Metacity (Marco)
    Steam Runtime Version:  steam-runtime-release_2015-06-12

Video Card:
    Driver:  NVIDIA Corporation GeForce GTX 960/PCIe/SSE2

    Driver Version:  4.5.0 NVIDIA 352.55
    OpenGL Version: 4.5
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 60 Hz
    VendorID:  0x10de
    DeviceID:  0x1401
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1280 x 1024
    Desktop Resolution: 1280 x 1024
    Primary Display Size: 13.31" x 10.63"  (17.01" diag)
                                            33.8cm x 27.0cm  (43.2cm diag)
    Primary Bus: PCI Express 16x
    Primary VRAM: 4096 MB
    Supported MSAA Modes:  2x 4x 8x 16x 

Sound card:
    Audio device: Realtek ALC892

Memory:
    RAM:  3953 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_US.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  929320 Mb
    Largest Free Hard Disk Block:  13796 Mb

Installed software:

Recent Failure Reports:  
```

**Comment by kisak-valve** [comment]
Closing as a duplicate of #898.
